app/facades/menu/show_facade.rb

```diff
@@ -8,10 +8,18 @@ def new_order
       @order = @order.decorate
     end
 
+    def placed_order?(current_user)
+      current_user.orders.find_by(menu_id: menu.id)
+    end
+
     def orders
       @orders ||= menu.orders.decorate
     end
 
+    def show_price?(current_user)
+      current_user.admin? && menu.orders.any?
```

Comment:
`exists?` is faster if you has `ActiveRecord::Relation`
[source](https://www.ombulabs.com/blog/benchmark/performance/rails/present-vs-any-vs-exists.html)
P.S. but you can check that one more time with benchmarks, because this article is 1 year old

Reply:
👍